OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -640,6 +681,35 @@ public void SetPosition(Actor self, WPos pos)
 				OnCruisingAltitudeReached();
 			else if (!isCruising && cruising)
 				OnCruisingAltitudeLeft();
+			FinishedMoving(self);
```

Comment:
I'd prefer an empty line above this for better readability.

Reply:
Fixed.